aviary/subsystems/propulsion/hamilton_standard.py

```
@@ -0,0 +1,1021 @@
+                            DMN = inputs['mach'][0] - ZMCRT
+                        else:
+                            ZMCRT = mach_tip_corr_arr[CL_tab_idx]
+                            DMN = inputs['tip_mach'][0] - ZMCRT
+                        XFFT[kl] = 1.0  # compressibility tip loss factor
+                        if (DMN > 0.0):
+                            CTE2 = CT_Eff*TXCLI[kl]*TBL
+                            XFFT[kl], run_flag = _biquad(comp_mach_CT_arr, 1, DMN, CTE2)
+                        CL_tab_idx = CL_tab_idx + 1
+                    if (CL_tab_idx_flg != 1):
+                        cli = inputs[Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICENT][0]
+                        TCLII, run_flag = _unint(
+                            CL_arr[CL_tab_idx_begin:CL_tab_idx_begin+4], TXCLI[CL_tab_idx_begin:CL_tab_idx_begin+4], cli)
+                        xft, run_flag = _unint(
+                            CL_arr[CL_tab_idx_begin:CL_tab_idx_begin+4], XFFT[CL_tab_idx_begin:CL_tab_idx_begin+4], cli)
+                    else:
+                        TCLII = TXCLI[CL_tab_idx_begin]
+                        xft = XFFT[CL_tab_idx_begin]
+                    ct = CTG[il]
+                    CT_Eff = CTG[il]*AFCTE*TCLII
+                    CTG1[il] = CT_Eff - CTTT[ibb]
+                    if (abs(CTG1[il]/CTTT[ibb]) < 0.001):
+                        ifnd1 = 1
+                        break
+                    if (il > 0):
+                        CTG[il+1] = -CTG1[il-1] * \
+                            (CTG[il] - CTG[il-1])/(CTG1[il] - CTG1[il-1]) + CTG[il-1]
+                        if (CTG[il+1] <= 0):
+                            ifnd2 = 1
+                            break
+
+                if (ifnd1 == 0 and ifnd2 == 0):
+                    print(
+                        f"Integrated design cl adjustment not working properly for ct definition (ibb={ibb})")
```

should this raise an error instead of just printing?